docs/source/virtual_rainforest/core/data.md

```diff
@@ -0,0 +1,153 @@
+---
+jupytext:
+  formats: md:myst
+  main_language: python
+  text_representation:
+    extension: .md
+    format_name: myst
+    format_version: 0.13
+    jupytext_version: 1.13.8
+kernelspec:
+  display_name: vr_python3
+  language: python
+  name: vr_python3
+---
+
+# Adding and using data with the Virtual Rainforest
```

Comment:
For some reason I can't seem to find this page in the sidebar, only by searching

Reply:
Fixed